packages/dashboard/src/app/routes/_authenticated/_facets/components/facet-values-table.tsx

```
@@ -4,6 +4,8 @@ import { Button } from '@/vdb/components/ui/button.js';
 import { addCustomFields } from '@/vdb/framework/document-introspection/add-custom-fields.js';
 import { graphql } from '@/vdb/graphql/graphql.js';
 import { useUserSettings } from '@/vdb/hooks/use-user-settings.js';
+import { validatePerPageValue } from '@/vdb/utils/pagination.js';
+import { useUserSettings } from '@/vdb/hooks/use-user-settings.js';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useUserSettings'. Consider to delete it or rename it.

'useUserSettings' is defined here:


(lint/suspicious/noRedeclare)